src/compiler/diagnosticMessages.json

```diff
@@ -2872,26 +2872,30 @@
         "category": "Error",
         "code": 7015
     },
-    "Property '{0}' implicitly has type 'any', because its 'set' accessor lacks a type annotation.": {
+    "Property '{0}' implicitly has type 'any', because its set accessor lacks a parameter type annotation.": {
         "category": "Error",
         "code": 7016
     },
-    "Index signature of object type implicitly has an 'any' type.": {
+    "Property '{0}' implicitly has type 'any', because its get accessor lacks a return type annotation.": {

```

Comment:
We shouldn't re-number more errors than we have to (it'll create RWC noise, etc). Maybe move both of these to a different error code if we really want them to have adjacent numbers.